osu.Game/Skinning/Editor/SkinBlueprint.cs

```diff
@@ -124,7 +124,7 @@ protected override void Update()
 
         public override bool ReceivePositionalInputAt(Vector2 screenSpacePos) => drawable.ReceivePositionalInputAt(screenSpacePos);
 
-        public override Vector2 ScreenSpaceSelectionPoint => drawable.ScreenSpaceDrawQuad.Centre;
+        public override Vector2 ScreenSpaceSelectionPoint => drawable.ToScreenSpace(drawable.OriginPosition);
```

Comment:
Blocking momentarily as I need to double-check this. It looks to be regressing the anchor/origin change logic (no longer holds its exact location).

Reply:
Never mind, this is fine and correct.